src/mechanics/constitutive/LinearElastic.h

```diff
@@ -37,12 +37,12 @@ class LinearElastic : public MechanicsInterface<TDim>
         mC = CalculateC(mE, mNu, planeState);
     }
 
+    static MechanicsTangent CalculateC(double E, double Nu, ePlaneState planeState = ePlaneState::PLANE_STRESS);
+
```

Comment:
Hey thanks! Good that it is accessible from the outside now.

Reply:
:kiss: